demo/micropython-run.ts

```diff
@@ -5,20 +5,37 @@ import { ConsoleLogger, LogLevel } from '../src/utils/logging';
 import { bootromB1 } from './bootrom';
 import { loadUF2, loadMicropythonFlashImage } from './load-flash';
 import fs from 'fs';
+const minimist = require('minimist');
+
+const args = minimist(process.argv.slice(2), {
+  string: 'image',  // UF2 image to load; defaults to "rp2-pico-20210902-v1.17.uf2"
+  boolean: 'no_gdb',  // Do NOT start GDB server
+})
 
 const mcu = new RP2040();
 mcu.loadBootrom(bootromB1);
 mcu.logger = new ConsoleLogger(LogLevel.Error);
-loadUF2('rp2-pico-20210902-v1.17.uf2', mcu);
+
+var image_name: string;
```

Comment:
`var` is deprecated - please use `let` instead.

Also, JavaScript's convention is to use camel case - so please use `imageName` instead of `image_name`